_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/@rhd/rhdp-search/rhdp-search-sort-page.ts

```diff
@@ -16,22 +16,23 @@ export default class RHDPSearchSortPage extends HTMLElement {
         this._sortChange = this._sortChange.bind(this);
     }
 
-    template = `<p>
-        <span>Sort results by</span>
-        <select>
-        <option value="relevance">Relevance</option>
-        <option value="most-recent">Most Recent</option>
-        </select>
-        </p>`; 
+    template = `
+        <div class="rhd-c-select">
```

Comment:
I had to create this to mimic the PatternFly component that does not use a <select> element.   I will create a follow up ticket about the arrow not being clickable (as that is how it works on the eloqua form currently as well.)